pkg/controller/quota/quota.go

```diff
@@ -79,11 +79,13 @@ func EnsureQuotaRequest(req router.Request, resp router.Response) error {
 	quotaRequest := &adminv1.QuotaRequestInstance{
 		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: namespace},
 		Spec: adminv1.QuotaRequestInstanceSpec{
-			Resources: adminv1.Resources{
-				Containers: len(app.Containers),
-				Jobs:       len(app.Jobs),
-				Volumes:    len(app.Volumes),
-				Images:     len(app.Images),
+			Resources: adminv1.QuotaRequestResources{
+				BaseResources: adminv1.BaseResources{
+					Containers: len(app.Containers),
+					Jobs:       len(app.Jobs),
+					Volumes:    len(app.Volumes),
+					Images:     len(app.Images),
+				},
```

Comment:
This is an example of how downstream dependencies will embed and use BaseResources.